models/repo/search.go

```diff
@@ -5,20 +5,72 @@ package repo
 
 import "code.gitea.io/gitea/models/db"
 
+// Strings for sorting result
+const (
+	// only used for repos
+	SearchOrderByAlphabetically        db.SearchOrderBy = "owner_name ASC, name ASC"
```

Comment:
If it wasn't already present in the original code, I'd propose dropping the `Search-` prefix in this PR.
It only confuses without bringing any additional value.
As it exists already, I'll see that I open another refactoring PR once this one was merged.

Reply:
-> #31359